conngen/conngenmodule.h

```diff
@@ -53,15 +53,10 @@ class ConnectionGeneratorModule : public SLIModule
   const std::string name() const;
   const std::string commandstring() const;
 
-  class CGConnect_cg_i_i_D_lFunction : public SLIFunction
+  class CGConnect_cg_g_g_D_lFunction : public SLIFunction
```

Comment:
I would try and give this function a more descriptive name

Reply:
`CGConnect` will probably be merged with the normal `Connect` function at one point. For backward-compatibility, I'd rather leave it like this in this PR, which is only about refactoring.